kube/service.go

```diff
@@ -138,11 +133,17 @@ func NewClusterIPService(role *model.Role, headless bool, public bool, settings
 		}
 	}
 	if public {
-		externalIPs := "[ 192.168.77.77 ]"
 		if settings.CreateHelmChart {
-			externalIPs = "{{ .Values.kube.external_ips | toJson }}"
+			externalIPs := "{{ .Values.kube.external_ips | toJson }}"
+			spec.Add("externalIPs", externalIPs, helm.Block("if not .Values.services.loadbalanced"))
+			spec.Add("type", "{{ if .Values.services.loadbalanced }} LoadBalancer {{ else }} ClusterIP {{ end }}")
+		} else {
+			externalIPs := "[ 192.168.77.77 ]"
```

Comment:
This should be `[]string{"192.168.77.77"}` (the old code was wrong); I actually fixed it in #354 but that hasn't landed yet so we're in a race :grin:

Reply:
Ok, I'm going to take #354 for review because it looks like we changed things in the same area.